ModulemdTranslationHelpers/Utils.py

```diff
@@ -1,14 +1,97 @@
 import sys
+import logging
+import gi
 from babel.messages import Catalog, pofile
 from datetime import datetime
 from collections import defaultdict
 
-try:
-    import gi
-    gi.require_version('Modulemd', '2.0')
-    from gi.repository import Modulemd
-except ImportError:
-    sys.exit(77)
+gi.require_version('Modulemd', '2.0')
```

Comment:
> @sgallagher : This whole try/except block should be removed, actually. The sys.exit(77) thing is only intended for tests. If the translation helpers don't have python-libmodulemd, it should just outright fail.

I actually had done that to escape the error generated by `pycodestyle` saying all imports should be on top of the file.

Reply:
You can fix that with a `# noqa` line, which is appropriate in this case.